docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md

```diff
@@ -1,12 +1,15 @@
 ---
 title: How to create and edit Expectations based on domain knowledge, without inspecting data directly
+tag: [how-to, getting started]
+description: Create ExpectationConfigurations based on domain knowledge.
+keywords: [Expectations, Domain Knowledge]
 ---
 
-import Prerequisites from '../../guides/connecting_to_your_data/components/prerequisites.jsx'
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
```

Comment:
Just need to add the technical tag import back in, they aren't getting rendered: 
![image](https://github.com/great-expectations/great_expectations/assets/9903066/d8935c98-2934-4a9f-ac05-974a0ea6e91a)
 https://deploy-preview-7852--niobium-lead-7998.netlify.app/docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly

Reply:
Thank you for this one.